network/httpd_proxy.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python
```

Comment:
Should be '#!/usr/bin/python' as per http://docs.ansible.com/ansible/developing_modules.html#module-checklist